src/features/localization/helpers/localization.helpers.ts

```diff
@@ -32,6 +32,10 @@ export const changeLanguage = (locale: string) => {
   updateLocalization();
 };
 
+export const getCurrentLanguage = () => {
```

Comment:
not sure we are using this yet! Also not sure if languages[0] will always return the current language.

Reply:
Fixed. https://www.i18next.com/overview/api#language